spec/features/groups/groups_index_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.feature 'Groups Index Page', type: :feature do
```

Comment:
- [ ] All your feature specs in the `spec/features` folder are failing. I think this has to do with selenium configuration or so. If the specs are passing on your end kindly add a screenshot so that subsequent reviewers can see it.

  ![image](https://github.com/Felipe-Perez-Ferraro/trans_by/assets/81468675/3857d6e6-c551-4c3b-ab7d-afe0eca5ceb5)